account_payment_other_company/views/account_payment.xml

```diff
@@ -0,0 +1,35 @@
+<odoo>
+
+    <!-- Copyright (C) 2019 Open Source Integrators
+         License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+    <!-- From an Invoice -->
+    <record id="account_payment_paid_by_invoice" model="ir.ui.view">
+        <field name="name">account.payment.tree.paid.by.invoice</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_invoice_form"/>
+        <field name="arch" type="xml">
+            <field name="journal_id" position="after">
+                <field name="company_id"/>
```

Comment:
```suggestion
```

Reply:
@max3903 The field needs to be there because it is being used in the domain, should I just make it invisible instead?